src/codegen/compile.ml

```diff
@@ -1387,21 +1387,21 @@ module Variant = struct
 
   *)
 
-  let tag_field = Tagged.header_size
-  let payload_field = Int32.add Tagged.header_size 1l
+  let variant_tag_field = Tagged.header_size
```

Comment:
I understood that by putting these identifiers into a `module Variant` namespace we don't need the `variant_` prefix. But I am not requesting a revert as @nomeata is fine with this.

Reply:
Even in this namespace we have two notions of tag so I think this clarifies things.